testutil/testnode/full_node.go

```diff
@@ -0,0 +1,124 @@
+package testnode
+
+import (
+	"os"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/cmd/celestia-appd/cmd"
+	pruningtypes "github.com/cosmos/cosmos-sdk/pruning/types"
+	"github.com/cosmos/cosmos-sdk/server"
+	srvtypes "github.com/cosmos/cosmos-sdk/server/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	"github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/libs/log"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+	"github.com/tendermint/tendermint/node"
+	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/privval"
+	"github.com/tendermint/tendermint/proxy"
+	dbm "github.com/tendermint/tm-db"
+)
+
+// New creates a ready to use tendermint node that operates a single validator
+// celestia-app network. The passed account names in fundedAccounts are used to
+// generate new private keys, which are included as funded accounts in the
+// genesis file. These keys are stored in the keyring that is returned in the
+// client.Context. NOTE: the forced delay between blocks, TimeIotaMs in the
+// consensus parameters, is set to the lowest possible value (1ms).
+//
+// note: the passed application config is currently unused atm, but we plan to
+// add support.
```

Comment:
[question] which parameter corresponds to the application config?

If present, proposal to explicitly include the parameter name in the note (because I don't think it's `tmCfg`).
If no longer present, proposal to remove this note.

Reply:
good catch 14aca810e103b3fbf803f0acadecbc3f234a999c